sw.js

```diff
@@ -4,7 +4,7 @@ const cacheName = 'fmb-kalimi-cache-2.2.25';
 const startPage = 'https://kalimijamaatpoona.org/fmb/';
 const offlinePage = 'https://kalimijamaatpoona.org/fmb/';
 const filesToCache = [startPage, offlinePage];
-const neverCacheUrls = [/\/users\/viewmenu.php/,/\/admin/];
+const neverCacheUrls = [/\/users\/viewmenu.php/,/\/users\/stopthali.php/,/\/users\/src\/custom.js/,/\/admin/];
```

Comment:
Remove this caching unless it is static content, as we dont know the impact of it.